homeassistant/components/calendar/google.py

```diff
@@ -45,18 +45,21 @@ class GoogleCalendarEventDevice(CalendarEventDevice):
     def __init__(self, hass, calendar_service, calendar, data):
         """Create the Calendar event device."""
         self.data = GoogleCalendarData(calendar_service, calendar,
-                                       data.get('search', None))
+                                       data.get('search', None),
+                                       data.get('transparency', None))
         super().__init__(hass, data)
 
 
 class GoogleCalendarData(object):
     """Class to utilize calendar service object to get next event."""
 
-    def __init__(self, calendar_service, calendar_id, search=None):
+    def __init__(self, calendar_service, calendar_id, search=None,
+                 transparency=True):
```

Comment:
Setting a default for `transparency` here doesn't make sense if you're defaulting to `None` above, does it?

Reply:
Well spotted and thanks again. I'll fix it in a minute.